helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml

```diff
@@ -63,10 +63,10 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    createdAt: "2023-08-17T13:36:51Z"
-    operators.operatorframework.io/builder: operator-sdk-v1.31.0
+    createdAt: "2023-09-06T18:13:45Z"
+    operators.operatorframework.io/builder: operator-sdk-v1.28.0-ocp
```

Comment:
Q: Did we downgrade the builder version?

Reply:
Sorry no, that wasn't intended.